scripts/src/java/org/oppia/android/scripts/common/BazelClient.kt

```diff
@@ -171,17 +163,8 @@ class BazelClient(private val rootDirectory: File, private val commandExecutor:
     return null
   }
 
-  /**
-   * Reads the content of the .dat file as a list of lines.
-   *
-   * @param filePath path to the .dat file
-   * @return content of the .dat file as a list of strings
-   */
-  private fun readDatFile(filePath: String?): List<String>? {
-    return filePath?.let { path ->
-      val pathObj = Paths.get(path)
-      File(pathObj.toUri()).readLines()
-    }
+  private fun readDatFile(filePath: String): List<String> {
```

Comment:
Given how simple this is, I suggest just inlining it and removing the helper at this point.

Reply:
Sure, Done!